src/hal/src/pso/mod.rs

```diff
@@ -206,21 +215,20 @@ pub struct Specialization {
 }
 
 /// Scalar specialization constant with value for overriding.
-/// DOC TODO, don't know what this means.
 #[derive(Debug, Clone)]
 pub enum Constant {
-    ///
+    /// `bool` value.
```

Comment:
heh, might want to just add `missing_docs` here, it's fairly self-descriptive

Reply:
👍